app/Http/Controllers/Api/V1/UserController.php

```diff
@@ -325,7 +325,8 @@ public function destroy(User $user)
     public function listNotifications(Request $request)
     {
         return response([
-            'notifications' => NotificationListResource::collection(auth()->user()->unreadNotifications),
+            'notifications' => $this->userRepository->fetchListing(auth()->user()->notifications()),
+            'is_unread_exist' => auth()->user()->unreadNotifications->count()
```

Comment:
perhaps the index should be 'unread_count', as you're not actually returning a boolean.
Just a suggestion here.